src/ebmc/ebmc_parse_options.cpp

```diff
@@ -245,58 +246,57 @@ void ebmc_parse_optionst::help()
+    " {y-I} {upath}                  \t set include path\n"
+    " {y--reset} {uexpr}             \t set up module reset\n"
     "\n"
     "Methods:\n"
-    " --k-induction                     do k-induction with k=bound\n"
-    " --bdd                             use (unbounded) BDD engine\n"
-    " --ic3 [options]                   use IC3 engine with options described below\n"
-    "       --property <nm>                  check property named <nm>\n"
-    "       --constr                         use constraints specified in 'file.cnstr'\n"
-    "       --h                              print out help information\n"
-    "       --new-mode                       new mode is switched on\n"
-    "       --aiger                          print out the instance in aiger format\n"
+    " {y--k-induction}               \t do k-induction with k=bound\n"
+    " {y--bdd}                       \t use (unbounded) BDD engine\n"
+    " {y--ic3}                       \t use IC3 engine with options described below\n"
+    "       {y--constr}              \t use constraints specified in 'file.cnstr'\n"
+    "       {y--new-mode}            \t new mode is switched on\n"
+    "       {y--aiger}               \t print out the instance in aiger format\n"
     
-    //" --interpolation                   use bit-level interpolants\n"
-    //" --interpolation-word              use word-level interpolants\n"
-    //" --diameter                        perform recurrence diameter test\n"
+    //" --interpolation                \t use bit-level interpolants\n"
+    //" --interpolation-word           \t use word-level interpolants\n"
+    //" --diameter                     \t perform recurrence diameter test\n"
     "\n"
     "Solvers:\n"
-    " --aig                             bit-level SAT with AIGs\n"
-    " --dimacs                          output bit-level CNF in DIMACS format\n"
-    " --smt1                            output word-level SMT 1 formula\n"
-    " --smt2                            output word-level SMT 2 formula\n"
-    " --boolector                       use Boolector as solver\n"
-    " --cvc4                            use CVC4 as solver\n"
-    " --mathsat                         use MathSAT as solver\n"
-    " --yices                           use Yices as solver\n"
-    " --z3                              use Z3 as solver\n"
+    " {y--aig}                       \t bit-level SAT with AIGs\n"
+    " {y--dimacs}                    \t output bit-level CNF in DIMACS format\n"
+    " {y--smt1}                      \t output word-level SMT 1 formula\n"
+    " {y--smt2}                      \t output word-level SMT 2 formula\n"
+    " {y--boolector}                 \t use Boolector as solver\n"
+    " {y--cvc4}                      \t use CVC4 as solver\n"
+    " {y--mathsat}                   \t use MathSAT as solver\n"
+    " {y--yices}                     \t use Yices as solver\n"
+    " {y--z3}                        \t use Z3 as solver\n"
```

Comment:
Is this supposed to be the same list of solvers as CBMC supports? Should it be the same list?

Reply:
No, this is quite separate